src/main/java/com/mide/gangsaeng/board/BoardResponse.java

```diff
@@ -0,0 +1,16 @@
+package com.mide.gangsaeng.board;
+
+import java.time.LocalDateTime;
+
+import lombok.Builder;
+import lombok.Getter;
+
+@Getter
+@Builder
+public class BoardResponse {
+    private String title;
+    private String content;
+    private int userId;
+    private LocalDateTime updatedAt;
```

Comment:
Will you not return id and createdAt?

Reply:
At first, I thought it wasn't necessary, but I decided to include it depending on the situation.

Reply:
Then your system cannot show when the post is written?

Reply:
I intended to show only the updated time, but I decided to **include both**(created time and updated time) because it seemed better.